run/authentication/auth_test.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This test deploys a secure application running on Cloud Run
+# to test that the authentication sample works properly.
+
+import auth
+
+import os
+import subprocess
+from urllib import request
+import uuid
+
+import pytest
+
+
+@pytest.fixture()
+def service():
+    # Unique suffix to create distinct service names
+    suffix = uuid.uuid4().hex
+    project = os.environ['GOOGLE_CLOUD_PROJECT']
+
+    # Deploy a hello-world Cloud Run Service
+    subprocess.run(
+        [
+            "gcloud",
+            "run",
+            "deploy",
+            f"helloworld-{suffix}",
+            "--image",
+            "gcr.io/gcbdocs/hello",
```

Comment:
Where did this image come from? Might want to add a comment above as to where to find the image source in case someone deletes the public image.

Maybe link to a source? Not sure where.
https://github.com/GoogleCloudBuild/code-examples/blob/master/quickstart-deploy/cloudbuild.yaml

Reply:
Nice catch on this! Given this is a serverless sample, maybe `gcr.io/cloudrun/hello` would be more appropriate, with a link back to https://github.com/GoogleCloudPlatform/cloud-run-hello

Reply:
Updated to cloudrun/hello.